crates/proto/src/xfer/dnssec_dns_handle.rs

```diff
@@ -352,16 +352,14 @@ where
             //       on a validation failure?
             // any error, is an error for all
             Err(e) => {
-                if log::log_enabled!(log::Level::Debug) {
```

Comment:
in this case we still want the enabled check, does tracing have that? I skips the message construction if debug is disabled, otherwise this might have a perf impact even though debug is disabled.